src/WrapperApp/components/Simulation/SimulationStatus.tsx

```diff
@@ -74,6 +74,36 @@ export default function SimulationStatus({ simulation, loadResults }: Simulation
 		loadResults?.call(null, null);
 	};
 
+	const onClickShowError = () => {
+		const errorWindow = window.open();
+
+		if (!errorWindow) return console.error('Could not open new window');
+		errorWindow.document.open();
+		errorWindow.document.write(
+			`<html>
+			<head>
+				<title>Error Details</title>
+			</head>
+			<body>
+			<h1>Details</h1>			
+			<h2>Input Files</h2>
+			<h3>beam.dat</h3>
```

Comment:
Do we need to hardcode filenames here ?
Can't this code simply make a loop over a dictionary (alphabetically sorted with keys) and show the files ?

Reply:
Good idea